packages/flutter_test/lib/src/matchers.dart

```diff
@@ -1041,6 +1041,10 @@ class _IsMethodCall extends Matcher {
 /// [RenderClipPath].
 const Matcher clipsWithBoundingRect = _ClipsWithBoundingRect();
 
+/// Asserts that a [Finder] locates a single object whose root RenderObject
+/// is not a [ClipRect], [ClipRRect], [ClipOval], or [ClipPath].
```

Comment:
the root render object is always a RenderView, and in any case none of the classes you list there are render objects, so I'm not sure this documentation is what you mean?